app/controllers/users_controller.rb

```diff
@@ -1,3 +1,24 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find_by(params[:id])
```

Comment:
```suggestion
    @user = User.find_by id: params[:id]
```
check trường hợp k tìm thấy @user

Reply:
Khi không tìm thấy @user
* Dùng find: Xảy ra exception ActiveRecord::RecordNotFound 
* Dùng find_by: Xảy ra exception NoMethodError